packages/client/src/comp/WarningMsg.test.tsx

```diff
@@ -1,20 +1,53 @@
 import React from 'react'
-import { render, fireEvent, act } from '@testing-library/react'
+import { render, fireEvent, act, wait } from '@testing-library/react'
 import { WarningMsg } from './WarningMsg'
 import { UnstatedContainer } from "./StateContainer"
+import { MemoryRouter } from 'react-router-dom'
+import { Initialize } from './Initialize'
+import { mocked } from 'ts-jest/utils'
+import { client } from '../lib/trpc'
+jest.mock('../lib/trpc.ts')
+jest.mock('../lib/analytics.ts')
+
+const renderResult = async () => {
+  const utmFields = {
```

Comment:
I don't think these are necessary for testing `WarningMsg`.  In general, there's no guarantee that any will be passed in